packages/compass-aggregations/src/components/focus-mode/focus-mode-stage-preview.tsx

```diff
@@ -86,10 +92,24 @@ export const FocusModePreview = ({
   stageIndex = -1,
   stageOperator = '',
   isMissingAtlasOnlyStageSupport = false,
+  expandPreviewDocsForStage,
+  collapsePreviewDocsForStage,
```

Comment:
Let's try to stick with React conventions here, callbacks usually named with an `on` prefix. Imagine this component outside of redux store and state, this callback is just called `on expand` (when expand action is clicked) or `on collapse`, it doesn't actually does the action